lib/chef/provider.rb

```diff
@@ -204,26 +204,39 @@ def converge_if_changed(*properties, &converge_block)
         specified_properties = properties.select { |property| new_resource.property_is_set?(property) }
         modified = specified_properties.select { |p| new_resource.send(p) != current_resource.send(p) }
         if modified.empty?
-          Chef::Log.debug("Skipping update of #{new_resource.to_s}: has not changed any of the specified properties #{specified_properties.map { |p| "#{p}=#{new_resource.send(p).inspect}" }.join(", ")}.")
+          properties_str = if @sensitive
```

Comment:
Please use `sensitive` rather than `@sensitive`, so that we can change up how things are implemented in the future.